app/models/post.rb

```diff
@@ -1,2 +1,3 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] A post also has an association with comments and likes, Please make sure to fix this issue so associations between models work as expected.
  - You can consider using this code.
```ruby
 has_many :comments
 has_many :likes